build/lite/charts/charts.go

```diff
@@ -34,8 +48,6 @@ func (charts *Charts) ListImages(clusterName string) ([]string, error) {
 			list = append(list, images...)
 		}
 	}
-
-	list = removeDuplicate(list)
```

Comment:
Why delete this?

Reply:
Image pull will be de duplicated. Here, de duplication is equivalent to two operations. The actual pull is to parse the merge data of the helm chart and read the image. The de duplication here only deals with the data of helm chart